docs/content/concepts/assets/asset-observations.mdx

```diff
@@ -5,7 +5,7 @@ description: Dagster provides functionality to record metadata about assets.
 
 # Asset Observations
 
-An asset observation is an event that records metadata about a given asset. Unlike asset materializations, asset observations do not signify that an asset has been mutated.
+You may wish to record metadata about an asset outside of that asset's materialization function. To do this, you can write another function and log <PyObject object="AssetObservation" /> events. An asset observation is an event that records metadata about a given asset. Unlike asset materializations, asset observations do not signify that an asset has been mutated.
```

Comment:
i felt like the intro was missing some motivation on why you would use an AssetObservation (rather than just log metadata in the asset materialization function). I still sort of feel that way, but this felt like a small improvement

Reply:
tbh me too. I think it's mostly helpful for reporting metadata about an asset after it's been materialized, which can be helpful if your asset calls out to an external tool that handles storing data for you (rather that doing IO managers). However, it looks like a lot of the content in the page is pretty old (like from right around when assets were first introduced), so it might be due for a refresh to account for all the changes since then